climada/hazard/base.py

```diff
@@ -265,73 +275,82 @@ def set_raster(self, files_intensity, files_fraction=None, attrs=None,
         if files_fraction is not None and len(files_intensity) != len(files_fraction):
             raise ValueError('Number of intensity files differs from fraction files: %s != %s'
                              % (len(files_intensity), len(files_fraction)))
-        self.tag.file_name = str(files_intensity) + ' ; ' + str(files_fraction)
+        haz = cls(haz_type, pool)
```

Comment:
I am not convinced by this change. Here you would overwrite the `haz_type` of sub-classes of `Hazard`.

Reply:
Proposal 65b654563e0faabdfd161ad4713f1776e16b5948